css/style.css

```diff
@@ -1,19 +1,167 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+/* HEADER */
+.header {
+    display: flex;
+}
+
+.header-image {
+    height: 50px;
+    width: 50px;
+    padding-left: 160px;
+    padding-top: 15px;
+}
+
+/* NAVIGATION */
+.nav-list {
+    display: flex;
+    list-style: none;
+    margin-left: 900px;
+}
+
+.nav-item {
+    padding: 17px;
+}
+
+.nav-link:hover {
+    color: #eb6728;
+}
+
+.nav-link {
+   text-decoration: none; 
+   color: #888;
+}
+
+/* BACKGROUND IMAGE, TEXT AND BUTTON */
+.page-top-half {
+    position: relative;
+}
 
+.page-top-half .large-text {
+    position: absolute;
+    z-index: 999;
+    margin: 0 auto;
+    left: 0;
+    right: 0; 
+    color: whitesmoke;
+    text-align: center; 
+}
+
+.page-top-half h1 {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    font-size: 60px;
+    top: 34%;
+    font-weight: 100;
+}
+
+.page-top-half p {
+    font-size: 34px;
+    top: 45%;
+    font-weight: 100;
+}
+
+.background-image {
+    width: 100%;
+} 
+
+.page-top-half button {
+    position: absolute;
+    left: 46%;
+    top: 59%;
+    padding: 15px 40px;
+    background-color: #eb6728;
+    color: white;
+    border: none;
+    border-radius: 4px;
+}
+
+/* BOTTOM HALF OF PAGE STYLING */
+.page-bottom-half {
+    margin-top: 150px;
+    margin-bottom: 90px;
+    text-align: center;
+}
+
+.page-bottom-half h1 {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    font-size: 60px;
+    font-weight: 300;
+    color: #3c3b3b;
+}
+
+.icon-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
```

Comment:
wonderful use of flex-box! By default `flex-direction` is set to `row` so you don't need to provide the property unless you're overriding where it was set by another selector